src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -575,7 +575,11 @@ export default {
 			return this.$store.getters.getActorType() === 'guests'
 		},
 
-		isMyMsg() {
+		isOwnMessage() {
+			if (this.conversation?.remoteToken) {
+				return this.actorId === this.$store.getters.getActorId() + '@' + window.location.host
+					&& this.actorType === ATTENDEE.ACTOR_TYPE.FEDERATED_USERS
+			}
```

Comment:
This will probably be used in many other places including : 
showSentIcon() 
isOwnMessageQuoted()